02/rguevara84/game.py

```diff
@@ -0,0 +1,102 @@
+#!python3
+# Code Challenge 02 - Word Values Part II - a simple game
+# http://pybit.es/codechallenge02.html
+
+import itertools
+import random
+
+from data import DICTIONARY, LETTER_SCORES, POUCH
+
+NUM_LETTERS = 7
+
+
+def draw_letters():
+    """Pick NUM_LETTERS letters randomly. Hint: use stdlib random"""
+    letters = random.sample(POUCH, k=NUM_LETTERS)
+    return letters
+
+def input_word(draw):
+    """Ask player for a word and validate against draw.
+    Use _validation(word, draw) helper."""
+    word = input().lower().strip()
+    if _validation(word, draw):
+        return word
+    else:
+        exit()
+    return word
+
+def _validation(word, draw):
+    """Validations: 1) only use letters of draw, 2) valid dictionary word"""
+    for letter in word:
+        if letter.upper() not in draw:
+            print("only use letters of draw")
+            raise(ValueError)
+        else:
+            state = True
+
+    if word not in DICTIONARY:
+        print ("Word not in dictionary")
```

Comment:
you don't need a `print` statement, each error class raised by `raise` can have a message that gets printed if the error is thrown: `raise ValueError("something went wrong...")`